src/components/VideoCard.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <article class="card">
-    <div class="card__bg" v-bind:style="{ 'background-image': 'url(' + info.pictures.sizes[2].link + ')' }"></div>
+    <div class="card__bg" v-bind:style="{ 'background-image': 'url(' + getImageBySize(info.pictures.sizes, 568, 170) + ')' }"></div>
```

Comment:
This `568` number is one of the responsive sizes from Foundation.

If this app were to scale, I'd use VueX or some other kind of centralised store, so it can be reused elsewhere without re-defining it on a per-component basis.